vnext/Microsoft.ReactNative/Modules/AppStateModule.cpp

```diff
@@ -63,12 +89,24 @@ void AppState::RemoveListeners(double /*count*/) noexcept {
 }
 
 ReactNativeSpecs::AppStateSpec_Constants AppState::GetConstants() noexcept {
-  return {m_active ? "active" : "background"};
+  return {m_enteredBackground ? "background" : m_deactived ? "inactive" : "active"};
```

Comment:
use parentheses when using nested ternaries (or rewrite as ifs)

Reply:
Thanks for pointing out the readability aspect!